code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -364,6 +364,27 @@
 		M.bodytemperature = max(310, M.bodytemperature - (8 * TEMPERATURE_DAMAGE_COEFFICIENT)) //310 is the normal bodytemp. 310.055
 	..()
 
+/datum/reagent/consumable/pwr_game
+	name = "Pwr Game"
+	description = "The only drink with the PWR that true gamers crave."
+	id = "pwr_game"
+	color = "#9385bf" // rgb: 58, 52, 75
```

Comment:
lemon lime should be verbatim what the guy from deus ex says, further adding to the reference.